internal/automation-manager/web-app/src/main/resources/global.properties

```diff
@@ -0,0 +1,13 @@
+automation.manager.host=localhost
+automation.manager.port=9766
+automation.manager.https-port=8243
+static-scanner.start-scan=am/1.0.0/staticScanner/startScan
+dynamic-scanner.start-scan=am/1.0.0/dynamicScanner/startScan
+get.static.scanners=am/1.0.0/getStaticScanners
+get.dynamic.scanners=am/1.0.0/getDynamicScanners
+webapp.client-id=04K_et0E9wyIKgMFOqFfGSDO6ska
+webapp.client-secret=OhH094zA7CizfpCfOTBO63CMwa4a
```

Comment:
Use placeholders for clientid and secret

Reply:
Replaced hard coded values with placeholders